rel/i18n/emqx_rule_engine_message_validation_schema.hocon

```diff
@@ -0,0 +1,84 @@
+emqx_rule_engine_message_validation_schema {
+
+  check_avro_type.desc:
+  """Avro schema check"""
+  check_avro_type.label:
+  """Avro schema check"""
+
+  check_avro_schema.desc:
+  """Schema name to use during check."""
+  check_avro_schema.label:
+  """Schema name"""
+
+  check_json_type.desc:
+  """JSON schema check"""
+  check_json_type.label:
+  """JSON schema check"""
+
+  check_json_schema.desc:
+  """Schema name to use during check."""
+  check_json_schema.label:
+  """Schema name"""
+
+  check_protobuf_type.desc:
+  """Protobuf schema check"""
+  check_protobuf_type.label:
+  """Protobuf schema check"""
+
+  check_protobuf_schema.desc:
+  """Schema name to use during check."""
+  check_protobuf_schema.label:
+  """Schema name"""
+
+  check_protobuf_message_name.desc:
+  """Message name to use during check."""
+  check_protobuf_message_name.label:
+  """Message name"""
+
+  check_sql_type.desc:
+  """SQL schema check"""
```

Comment:
Maybe describe this in more detail.
e.g. 
```
Use rule-engine's SQL to validate the message. SQL here is the same as in rule-engine,
just that there is no need for the `FROM` clause.
An SQL statement which yields any value is considered a successful validation, otherwise failed.
For example <code>SELECT payload.foo + payload.bar as sum WHER sum > 0</code>
validates that the sum of field `foo` and `bar` is a positive value.
```